src/cosmos/authz/v1beta1/tx.ts

```diff
@@ -1,12 +1,7 @@
-/* eslint-disable */
-import Long from "long";
-import * as _m0 from "protobufjs/minimal";
-import { Grant } from "../../../cosmos/authz/v1beta1/authz";
+import { Grant } from "./authz";
 import { Any } from "../../../google/protobuf/any";
-
-export const protobufPackage = "cosmos.authz.v1beta1";
```

Comment:
let me double check — I think the reason I got rid of these is so you can do `export * from '...'` for many of these modules.

I think because the variable name is the same it created collisions.

However, I can still test out adding these as and using scoped bundles, and in that case I believe this could be added back.

Reply:
here is a PR of the plugin https://github.com/osmosis-labs/telescope/pull/128/files

here is the generated types https://github.com/osmosis-labs/telescope/pull/129/files